src/test/java/org/openrewrite/java/testing/assertj/AdoptAssertJDuractionAssertionsTest.java

```diff
@@ -0,0 +1,426 @@
+              
+              class Foo {
+                  void testMethod(Temporal timestampA, Temporal timestampB) {
+                      assertThat(Duration.between(timestampA, timestampB)).hasNanos(1);
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    //millis to seconds
+    @Test
+    void millisToSeconds() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import java.time.Duration;
+              
+              import static org.assertj.core.api.Assertions.assertThat;
+              
+              class Foo {
+                  void testMethod(Duration time) {
+                      assertThat(time).hasMillis(5000);
+                  }
+              }
+              """,
+            """
+              import java.time.Duration;
+              
+              import static org.assertj.core.api.Assertions.assertThat;
+              
+              class Foo {
+                  void testMethod(Duration time) {
+                      assertThat(time).hasSeconds(5);
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    //seconds to minutes
+    @Test
+    void secondsToMinutes() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import java.time.Duration;
+              
+              import static org.assertj.core.api.Assertions.assertThat;
+              
+              class Foo {
+                  void testMethod(Duration time) {
+                      assertThat(time).hasSeconds(600);
+                  }
+              }
+              """,
+            """
+              import java.time.Duration;
+              
+              import static org.assertj.core.api.Assertions.assertThat;
+              
+              class Foo {
+                  void testMethod(Duration time) {
+                      assertThat(time).hasMinutes(10);
+                  }
+              }
+              """
+          )
+        );
+    }
+
+
+
+    //millis to minutes
+    @Test
+    void millisToMinutes() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import java.time.Duration;
+              
+              import static org.assertj.core.api.Assertions.assertThat;
+              
+              class Foo {
+                  void testMethod(Duration time) {
+                      assertThat(time).hasMillis(300000);
+                  }
+              }
+              """,
+            """
+              import java.time.Duration;
+              
+              import static org.assertj.core.api.Assertions.assertThat;
+              
+              class Foo {
+                  void testMethod(Duration time) {
+                      assertThat(time).hasMinutes(5);
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    //millis to hours
```

Comment:
I don't think these comments add much; can we remove these?

Reply:
oh yes, I just had them there to help me when I was writing them all out